ts/modules/boost/wgl-renderer.ts

```diff
@@ -1645,7 +1645,7 @@ function GLRenderer(
         shader.destroy();
         if (gl) {
 
-            objectEach(textureHandles, function (key: string): void {
+            objectEach(textureHandles, function (_val, key): void {
```

Comment:
Why the dangling underscore? We have been trying to avoid those, though we have some of them for historic reasons.

Reply:
It is used in ESLint and TypeScript to mark unused function parameters, so that no complaints appear.